RecoLocalCalo/EcalRecAlgos/interface/EcalUncalibRecHitRecAnalFitAlgo.h

```diff
@@ -15,10 +15,14 @@
 #include <string>
 
 #include "TROOT.h"
-#include "TMinuit.h"
+//#include "TMinuit.h"
```

Comment:
I propose to clean the commented include here

Reply:
done